src/PKSim.UI/BootStrapping/ApplicationStartup.cs

```diff
@@ -69,6 +71,16 @@ public void InitializeForStartup()
 
          container.Register<IConfigurableContainerLayoutView, AccordionLayoutView>(ViewLayouts.AccordionView.Id);
          container.Register<IConfigurableContainerLayoutView, TabbedLayoutView>(ViewLayouts.TabbedView.Id);
+
+         configureLogger(container);
+      }
+
+      private void configureLogger(IContainer container)
+      {
+         var loggerFactory = container.Resolve<ILoggerFactory>();
+         loggerFactory
+            .AddPresenter()
+            .AddDebug();
```

Comment:
Also add the debug logger. That way we always see what's happening in the debug windown